src/instructlab/model/chat.py

```diff
@@ -61,6 +62,28 @@
 
 PROMPT_PREFIX = ">>> "
 
+DEFAULT_ENDPOINT = cfg.get_api_base(cfg._serve.host_port)
+
+
+def is_openai_server_and_serving_model(
```

Comment:
Where is ` is_openai_server_and_serving_model()` getting called? I don't see another call for it in the PR

Reply:
Ruh roh - looks like I may have accidentally deleted that change from this PR.